xml/art_admin_slemicro_selinux.xml

```diff
@@ -196,7 +196,7 @@
     <listitem>
      <para>
       conveys information how particular files and processes can interact. A
-      process is assigned files with a concrete SELinux type and it cannot
+      process is assigned files with a concrete SELinux type, and it cannot
```

Comment:
```suggestion
      process is assigned to files with a concrete SELinux type, and it cannot
```

Reply:
It seems my comment got lost, but a process is not assigned to files, but files are assigned to a process. The direction here is the opposite, then suggested by the correction.